src/test/java/com/efficio/fieldbook/web/util/WorkbookUtilTest.java

```diff
@@ -510,4 +512,19 @@ public void testGetVariatesUsedInFormulas() {
 		Assert.assertTrue(measurementVariableList.get(0).equals(measurementVariable2.getTermId()));
 
 	}
+
+	@Test
+	public void testCreateMeasurementRowsFromEnvironments() {
+		final MeasurementVariable variable =
+			MeasurementVariableTestDataInitializer.createMeasurementVariable(TermId.PLOT_CODE.getId(), TermId.PLOT_CODE.name(), null);
+		variable.setDataTypeId(TermId.NUMERIC_VARIABLE.getId());
+		final Instance instance = new Instance();
+		instance.setStockId(1);
```

Comment:
Can you use different test values for stock, experiment and instance IDs for better assertion?